examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -1090,8 +1090,8 @@ server cluster EthernetNetworkDiagnostics = 55 {
 }
 
 server cluster FixedLabel = 64 {
-  attribute(readonly) LabelStruct labelList[] = 0;
-  attribute(readonly) int16u clusterRevision = 65533;
+  attribute(readonly, reportable) LabelStruct labelList[] = 0;
```

Comment:
I think you will need to rerun zap to avoid this delta. Trying to fix it in #13632 where client and server reportable are missing.

I will add a checkmark though as this is just zap being unstable.

Reply:
I cannot get it to work.

Reply:
Applied the diff printed by CI. This is the only way I could avoid the other changes.